endpoints_management/control/wsgi.py

```diff
@@ -276,6 +276,7 @@ def __call__(self, environ, start_response):
         app_info.http_method = http_method
         app_info.url = parsed_uri
 
+        consumer_project_number = 0
```

Comment:
Mark the meaning of this constant? (Via a comment, probably)

Reply:
Done.